docs/import/manage_disaster.html

```diff
@@ -51,6 +51,15 @@
     <form action="" id="new-disaster-form">
       <label for="name">Disaster Name: </label><input id="name"><label for="name"><i> ex: harvey</i></label><br>
       <label for="year">Disaster Year: </label><input id="year"><label for="year"><i> ex: 2017</i></label><br>
+      <div>
+      Will you be using U.S. Census data for this disaster? If not, you will have to provide
```

Comment:
Is the real distinction here is the disaster US state-based or not? Like there is no situation where a disaster hits one of the 50 US states and the user shouldn't take the traditional approach right? It might be confusing for people who aren't familiar with the available data that 50 states disaster === US Census Data.

Reply:
Yes, but they might also use Census data for a U.S. territory, or they might not. Census data seemed like the actual criterion. Happy to rephrase. Do you prefer "Are you focused on this disaster's impact on U.S. states and/or territories with Census data available?" Seems unwieldy to me. Other suggestions?

Reply:
Yeah, honestly why i didn't leave a suggestion was because I couldn't think of something quippy. Maybe the answer is to link to a help that leads to a long section @ruthtalbot can write about what data we know exists at this point?

Reply:
SNAP data and TIGER shapefiles, I think.

Reply:
I added a link to the general help doc here. @ruthtalbot I think we need a higher-level discussion of when you would want a Census-based vs flexible disaster. Left a TODO for you here to add that.